services/convert/issue.go

```diff
@@ -44,7 +44,7 @@ func toIssue(ctx context.Context, issue *issues_model.Issue, getDownloadURL func
 	apiIssue := &api.Issue{
 		ID:          issue.ID,
 		Index:       issue.Index,
-		Poster:      ToUser(ctx, issue.Poster, nil),
+		Poster:      ToUser(ctx, issue.Poster, doer),
```

Comment:
Why "doer" could be "poster"?

Reply:
`doer` is the consumer of the data. `Poster` is the field of the issue. They maybe the same if one view the his own issues.